docs/content/_navigation.json

```diff
@@ -241,6 +241,10 @@
           {
             "title": "Asset Materializations",
             "path": "/concepts/assets/asset-materializations"
+          },
+          {
+            "title": "Software-Defined Assets",
+            "path": "/concepts/assets/software-defined-assets"
```

Comment:
thought on moving "ASSETS" section right below "OPS, JOBS, AND GRAPHS" - making it the second entry in the left nav

Reply:
interesting. I think I'm in favor, but I'd have to think about it a little more. going to leave that for a followup.